consensus/src/chained_bft/consensusdb/schema/single_entry/test.rs

```diff
@@ -1,13 +0,0 @@
-// Copyright (c) The Libra Core Contributors
-// SPDX-License-Identifier: Apache-2.0
-
-use super::*;
-use schemadb::schema::assert_encode_decode;
-
-#[test]
-fn test_single_entry_schema() {
-    assert_encode_decode::<SingleEntrySchema>(
-        &SingleEntryKey::ConsensusState,
```

Comment:
lol, you probably can just change the enum instead of deleting

Reply:
Oh, lemme poke at that again, I wasn't sure what this was testing :(